hw6_functional/functional.py

```diff
@@ -0,0 +1,80 @@
+## Task 1
+
+# This function takes unlimited number of functions and a collection of values as input.
+# It returns list of results of sequential application of these functions to the collection of values.
+
+def sequential_map(*args):
+    *functions, values = [*args]              # unpacking input functions and collection of values
```

Comment:
Эта конструкция немного избыточна
```suggestion
    *functions, values = args              # unpacking input functions and collection of values
```